tools/src/wyvern/tools/tests/EffectSystemTests.java

```diff
@@ -389,4 +389,95 @@ public void testObjNetwork() throws ParseException {
         /* Object with effect annotations. */
         TestUtil.doTestScriptModularly(PATH, "effects.objNetwork", Util.stringType(), new StringLiteral("ObjNetwork with effects"));
     }
+
+    @Test
+    public void testSeparation1() throws ParseException {
```

Comment:
This test suite checks the basic functionality of the effect system, i.e., when modules are fully annotated with effects. Since the added tests check something more than the basic effect-checking behavior, please create a new test suite for them and move them there.